README.md

```diff
@@ -193,19 +194,32 @@ For most users, setting `image: [path-to-image]` on a per-page basis should be e
 * `path` - The relative path to the image. Same as `image: [path-to-image]`
 * `twitter` - The relative path to a Twitter-specific image.
 * `facebook` - The relative path to a Facebook-specific image.
-* `height` - The height of the Facebook (`og:image`) image
-* `width` - The width of the Facebook (`og:image`) image
+* `height` - The height of the Facebook (`og:image`) image and JSON-LD image object.
+* `width` - The width of the Facebook (`og:image`) image and JSON-LD image object.
+
+The JSON-LD will default to the `image: ` tag unless `path: ` `height: ` and `width: `.
 
 You can use any of the above, optional properties, like so:
 
 ```yml
 image:
+  path: /img/banner.png
   twitter: /img/twitter.png
   facebook: /img/facebook.png
   height: 100
   width: 100
 ```
 
+Or if you have no site specific images simply:
+
+```yml
+image:
+  path: /img/banner.png
+  height: 100
+  width: 100
+```
```

Comment:
This is only if one wants to control size, correct? Otherwise they can just specify image as a path if I'm not mistaken. If that's true this is probably not needed as it may make the doc less approachable.